src/example-relay/src/pages/_document.js

```diff
@@ -34,6 +32,7 @@ export default class MyDocument extends Document {
             href="https://fonts.googleapis.com/css?family=Roboto:400,400i,500,500i,700"
             rel="stylesheet"
           />
+          <style dangerouslySetInnerHTML={{ __html: this.props.css }} />
```

Comment:
won't this add your style tag 2 times?

Reply:
Yes, it should not be there - rebase mistake.